tableofcontents.md

```diff
@@ -49,6 +56,8 @@ Explain
 * serial interval
 * generation time
 
+Please look up some paper how to estimate (i.e. which distribution are they regarded to follow?) these indexes above. 
+
```

Comment:
Started to wonder if it's useful to append 'advanced' tasks to some of the basic tasks? Looking up papers that estimate those intervals may be too much for beginners, but some eager learners may find it nice